File: Orders/orders/src/main/java/com/soa/orders/domain/models/Order.java
package com.soa.orders.domain.models;

import com.soa.orders.domain.models.enums.Status;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import java.util.List;

@Getter @Setter
public class Order {

    private Long id;

    private long total;

    private Date date;

    private Status status;

    private List<OrderProduct> products;

}

File: Inventory/src/domain/model/Order.ts
import { v4 as uuidv4 } from 'uuid';
import { OrderProduct } from './OrderProduct';

export class Order{
    public uuid: string;
    public total:number;
    public date: string;
    public status: string;
    public products: OrderProduct[]  = [];


    constructor(total:number,date:string,status:string,products:OrderProduct[]){
        this.uuid = uuidv4();
        this.total = total; 
        this.date = date;
        this.status = status;
        this.products = products;    
    }
}